metecho/api/serializers.py

```diff
@@ -377,7 +384,30 @@ def get_pr_url(self, obj) -> Optional[str]:
     def create(self, validated_data):
         validated_data.pop("should_alert_dev", None)
         validated_data.pop("should_alert_qa", None)
-        return super().create(validated_data)
+        dev_org = validated_data.pop("dev_org", None)
+        user = self.context["request"].user
+        originating_user_id = str(user.id)
+
+        if dev_org and not validated_data.get("assigned_dev"):
```

Comment:
The current user will only be assigned as Dev if not provided by the frontend. I figured in the future we might want to allow assigning a developer from the "Create Task" modal.